saleor/page/models.py

```diff
@@ -1,40 +1,31 @@
 import datetime
 
 from django.db import models
-from django.db.models import Q
 from django.urls import reverse
 from django.utils.translation import pgettext_lazy
 
 from ..core.utils import build_absolute_uri
 from ..core.utils.translations import TranslationProxy
+from ..product.models import PublishedQuerySet
 from ..seo.models import SeoModel, SeoModelTranslation
 
 
-class PageQuerySet(models.QuerySet):
+class PagePublishedQuerySet(PublishedQuerySet):
```

Comment:
I'm not a big fan of inheritance but I feel tempted to move `PublishedQuerySet` to `saleor.core.models` an abstracting `is_published` and `publication_date` fields to some kind of `PubilshableModel` abstract model. What do you think?

Reply:
That sounds good, I didn't do it to keep it simple, but that works as well 💃